ChangeLog

```diff
@@ -6,7 +6,7 @@ What's New in astroid 2.9.0?
 ============================
 Release date: TBA
 
-
+* Extended attrs brain to support the provisional APIs
```

Comment:
I'm not sure if this is supposed to go in 2.9, I think it could easily be marked as a patch for semver. I'll move it to 2.8.1 for now.